src/include/ipc4/base_fw.h

```diff
@@ -289,6 +289,11 @@ enum ipc4_basefw_params {
 	/* Use LARGE_CONFIG_SET to change SDW ownership */
 	IPC4_SDW_OWNERSHIP = 31,
 
+	/* This command is used by SW to notify FW for changing state of Mic Privacy
+	 * for SoundWire Gateways
+	 */
+	SNDW_MIC_PRIVACY_HW_MANAGED_STATE_CHANGE = 35,
```

Comment:
IPC4_SET_MIC_PRIVACY_FW_MANAGED_POLICY_MASK has a IPC4_ prefix, so why not IPC4_SNDW_MIC_PRIVACY_HW_MANAGED_STATE_CHANGE